packages/test/src/test-integration-workflows.ts

```diff
@@ -14,7 +14,8 @@ import {
 } from '@temporalio/worker';
 import * as activity from '@temporalio/activity';
 import * as workflow from '@temporalio/workflow';
-import { CancelReason } from '@temporalio/worker/src/activity';
+import { CancelReason } from '@temporalio/worker/lib/activity';
```

Comment:
We should export this from worker or activity if we don’t already

Reply:
ATM, that would have to be exported from 'worker'...

But why? We don't use that type on anything public, except for `CancelledFailure.message`, but we can't easily change the type of the `message` property (it's in common and may also be used in the context of a Workflow, which implies different set of values).

Reply:
Still useful for users to compare and I'd like to have a property on activity context for the cancel reason.

Reply:
Another thing... on deserializing a `CancelledFailure` object from an incoming failure (ie. failure-converter.ts), the incoming message may come from a different SDK, and could therefore contain any message (ie. not bound to `CancelReason`)

Reply:
We don't have any API that throws CancelledFailure directly.

Reply:
I don't think the message property should be typed, we use CancelledFailure all over the SDK.